docs/content/_index.md

```diff
@@ -1,16 +1,27 @@
 ---
-title: Brigade Documentation
+title: Brigade 2.0 Documentation
 description: Homepage of the Brigade Documentation
 ---
 
 # Brigade Documentation
 
-Everything you need to know about Brigade.  
+Everything you need to know about Brigade.
+
+If you haven't used Brigade before, welcome! 🎉
+Brigade 2.0 has a new design that makes learning and interacting with Brigade easier than ever before.
```

Comment:
nit: should this (what is new about 2.0) be the first thing new users see?
I assume most people looking at the getting started page would not be familiar with brigade, so would it be more helpful to describe here what Brigade is?